ci/release/update-version.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+###############################
+# Integration Version Updater #
+###############################
+
+## Usage
+# bash update-version.sh <new_version>
+
+
+# Format is YY.MM.PP - no leading 'v' or trailing 'a'
+NEXT_FULL_TAG=$1
+
+# Get current version
+CURRENT_TAG=$(git tag --merged HEAD | grep -xE '^v.*' | sort --version-sort | tail -n 1 | tr -d 'v')
+CURRENT_MAJOR=$(echo $CURRENT_TAG | awk '{split($0, a, "."); print a[1]}')
+CURRENT_MINOR=$(echo $CURRENT_TAG | awk '{split($0, a, "."); print a[2]}')
+CURRENT_PATCH=$(echo $CURRENT_TAG | awk '{split($0, a, "."); print a[3]}')
```

Comment:
```suggestion
CURRENT_PATCH=$(echo $CURRENT_TAG | awk '{split($0, a, "."); print a[3]}' | tr -d 'a')
```
I don't think this change has been propagated to all the repos yet, but you may want to add this as we've done [in cucim](https://github.com/rapidsai/cucim/blob/0f1fcb42f746f2ed58a9e6aff4ddc4dd553dc3df/ci/release/update-version.sh#L17) to remove the `a` if it exists.